packages/cli/src/command.ts

```diff
@@ -55,7 +54,7 @@ export abstract class Command<
     // Authenticate the Ceramic instance whenever a key is provided
     if (this.flags.key != null) {
       const did = await this.getAuthenticatedDID(this.flags.key)
-      this.spinner.info(`Using DID ${chalk.cyan(did.id)}`)
+      this.spinner.info(`Using DID ${did.id}`)
```

Comment:
Why this change please?

Reply:
Thanks for catching this. There was an issue with default imports that I was hunting & I removed chalk from here. will be readding shortly!